src/module/canvas/token/object.ts

```diff
@@ -615,6 +615,34 @@ class TokenPF2e<TDocument extends TokenDocumentPF2e = TokenDocumentPF2e> extends
         return super._onHoverOut(event);
     }
 
+    /** Require that a loot actor or dead creature is in reach for a player to view its sheet. */
+    protected override _onClickLeft2(event: PIXI.FederatedPointerEvent): void {
+        const actor = this.actor;
+        const requiresReach = game.pf2e.settings.automation.reachEnforcement.has(
+            actor?.isOfType("loot") ? (actor.isLoot ? "loot" : "merchants") : "corpses",
+        );
+        if (!requiresReach || this.document.isOwner || !actor?.isLootableBy(game.user) || !canvas.grid.isSquare) {
+            return super._onClickLeft2(event);
+        }
+        const isInReach = R.unique(
+            [canvas.tokens.controlled, game.user.character?.getActiveTokens(true, false) ?? []].flat(),
+        ).some(
+            (t) =>
+                t.actor?.isOwner &&
+                t.actor.isOfType("creature") &&
```

Comment:
this prevents Party tokens from counting, which seems bad for a group that uses a single shared Party token to move around a dungeon and open doors etc until they find an encounter.